scripts/dpop-verify.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
+#
+# Copyright 2023 The Sigstore Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Print all commands and stop on errors
+set -o errexit
+set -o xtrace
+set -u
+
+BRANCH=$(git rev-parse --abbrev-ref HEAD)
```

Comment:
right now this script will run correctly inside the GH workflow when called in the wrapper. is there a way you could structure this so that local users can also run this (with something like `./scripts/dpop-verify.sh ${PR}` (and have the script internally find the delegation role name and branch)

Feel free to file an issue for follow-up, let's get something working - local verification isn't bad with instructions to run step-0 and then the explicit commands.

Reply:
Yes, that is a verify good idea!

Reply:
perfect!